views/chooseAll-answer.pug

```diff
@@ -0,0 +1,45 @@
+if answerForm
+	.form-group.col.s12
+		each index in [1,2,3,4]
+			.row
+				.input-field.col.s1
+					input(type='checkbox', name='checkButton', value='mcans'+index, id='mcans'+index)
```

Comment:
why is the value mcans

Reply:
this applies to all of those in this file

Reply:
have removed it, not needed